<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <link rel="stylesheet" href="assets/css/starter.css" />
  <title>Team:Estonia_TUIT - 2020.igem.org</title>
</head>

<body>

  <!-- Navigation -->
  <nav class="navbar navbar-expand-lg navbar-light fixed-top mainNav">
    <script type="module" src="assets/js/navbar.js"></script>
    <div class="container">
      <a class="navbar-brand js-scroll-trigger" href="#page-top">
        <img src="https://2020.igem.org/wiki/images/b/b5/T--Estonia_TUIT--tuitlogo.png" alt="" height="66">
      </a>
      <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse"
        data-target="#navbarResponsive">
        <span class="navbar-toggler-icon"></span>
      </button>

      <div class="collapse navbar-collapse" id="navbarResponsive">
        <ul class="navbar-nav nav ml-auto">
          <li class="nav-item dropdown">
            <a class="nav-link dropdown-toggle" href="#" data-toggle="dropdown">Project</a>
            <div class="dropdown-menu fade-up" aria-labelledby="navbarDropdownMenuLink2">
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Description">Description</a>
              <div class="dropdown-divider"></div>
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Design">Design</a>
              <div class="dropdown-divider"></div>
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Contribution">Contribution</a>
              <div class="dropdown-divider"></div>
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Implementation">Implementation</a>
          </li>
          <li class="nav-item dropdown">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink2" data-toggle="dropdown"
              aria-haspopup="true" aria-expanded="false">Results</a>
            <div class="dropdown-menu fade-up" aria-labelledby="navbarDropdownMenuLink2">
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Results">Results</a>
              <div class="dropdown-divider"></div>
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Model">Modelling</a>
              <div class="dropdown-divider"></div>
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Proof_Of_Concept">Proof of Concept</a>
              <div class="dropdown-divider"></div>
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Engineering">Engineering</a>
          </li>
          <li class="nav-item dropdown">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink3" data-toggle="dropdown"
              aria-haspopup="true" aria-expanded="false">Human Practices</a>
            <div class="dropdown-menu fade-up" aria-labelledby="navbarDropdownMenuLink3">
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Human_Practices">Human Practices</a>
              <div class="dropdown-divider"></div>
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Partnership">Partnership</a>
              <div class="dropdown-divider"></div>
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Collaborations">Collaborations</a>
              <div class="dropdown-divider"></div>
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Education">Education</a>
          </li>
          <li class="nav-item dropdown">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink4" data-toggle="dropdown"
              aria-haspopup="true" aria-expanded="false">Lab</a>
            <div class="dropdown-menu fade-up" aria-labelledby="navbarDropdownMenuLink4">
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Parts">Parts</a>
              <div class="dropdown-divider"></div>
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Notebook">Notebook</a>
              <div class="dropdown-divider"></div>
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Experiments">Experiments</a>
              <div class="dropdown-divider"></div>
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Safety">Safety</a>
          </li>
          <li class="nav-item dropdown">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink4" data-toggle="dropdown"
              aria-haspopup="true" aria-expanded="false">People</a>
            <div class="dropdown-menu fade-up" aria-labelledby="navbarDropdownMenuLink4">
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Team">Team</a>
              <div class="dropdown-divider"></div>
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Attributions">Attributions</a>
              <div class="dropdown-divider"></div>
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Sponsors">Sponsors</a>
              <div class="dropdown-divider"></div>
              <a class="dropdown-item" href="https://2020.igem.org/Team:Estonia_TUIT/Videos">Videos</a>
          </li>
        </ul>
      </div>
    </div>
  </nav>

  <!-- collab banner -->
  <section class="banner collaborationsbanner" id="tsparticles">
    <script src="assets/js/tsparticles.min.js"></script>
    <script type="module" src="assets/js/sparkles.js"></script>
    <h1>Experiments</h1>
  </section>
  <!-- collab text and stuff -->  
  <section class="generic-section">
    <div class="flex-container">
      <div class="introcontainer">
        <h2 class="introduction">Introduction</h2>
      </div>
      <div class="collabtext item">Protocols play a key role in uniformity of processes and creation of successful products during
        experimental laboratory work. Protocols outlined here were provided to us by supervisors and sponsors
        and are the basis for all experiments mentioned in the Notebook.</div>
    </div>
  </section>

  <section class="carouselsection">
    <div class="carouselcontainer">
      <div id="experimentCarousel" class="carousel slide carousel-fade" data-interval="20000" data-ride="false">
          <ol class="carousel-indicators">
              <li data-target="#experimentCarousel" data-slide-to="0" class="active"><p>1</p></li>
              <li data-target="#experimentCarousel" data-slide-to="1"><p>2</p></li>
              <li data-target="#experimentCarousel" data-slide-to="2"><p>3</p></li>
              <li data-target="#experimentCarousel" data-slide-to="3"><p>4</p></li>
              <li data-target="#experimentCarousel" data-slide-to="4"><p>5</p></li>
              <li data-target="#experimentCarousel" data-slide-to="5"><p>6</p></li>
              <li data-target="#experimentCarousel" data-slide-to="6"><p>7</p></li>
              <li data-target="#experimentCarousel" data-slide-to="7"><p>8</p></li>
              <li data-target="#experimentCarousel" data-slide-to="8"><p>9</p></li>
              <li data-target="#experimentCarousel" data-slide-to="9"><p>10</p></li>
              <li data-target="#experimentCarousel" data-slide-to="10"><p>11</p></li>
              <li data-target="#experimentCarousel" data-slide-to="11"><p>12</p></li>
              <li data-target="#experimentCarousel" data-slide-to="12"><p>13</p></li>
          </ol>
          <ol class="carousel-indicators indicator-row2">
              <li data-target="#experimentCarousel" data-slide-to="13"><p>14</p></li>
              <li data-target="#experimentCarousel" data-slide-to="14"><p>15</p></li>
              <li data-target="#experimentCarousel" data-slide-to="15"><p>16</p></li>
              <li data-target="#experimentCarousel" data-slide-to="16"><p>17</p></li>
              <li data-target="#experimentCarousel" data-slide-to="17"><p>18</p></li>
              <li data-target="#experimentCarousel" data-slide-to="18"><p>19</p></li>
              <li data-target="#experimentCarousel" data-slide-to="19"><p>20</p></li>
              <li data-target="#experimentCarousel" data-slide-to="20"><p>21</p></li>
              <li data-target="#experimentCarousel" data-slide-to="21"><p>22</p></li>
              <li data-target="#experimentCarousel" data-slide-to="22"><p>23</p></li>
              <li data-target="#experimentCarousel" data-slide-to="23"><p>24</p></li>
              <li data-target="#experimentCarousel" data-slide-to="24"><p>25</p></li>
              <li data-target="#experimentCarousel" data-slide-to="25"><p>26</p></li>
            </ol>
          </ol>
          <div class="carousel-inner">
            <div class="carousel-item active">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">DNA polymerase chain reaction protocol</h3>
                <p class="exp-recipe-paragraph">To amplify DNA fragments for subsequent cloning, PCR with Phusion DNA polymerase was used. Components of PCR reaction are listed in <b>Table 1</b>.</p>
                <p class="exp-recipe-paragraph">All components of the PCR reaction were mixed on ice. PCR tubes with the reaction mix were transferred into a 98 °C preheated thermocycler and PCRs were performed according to the conditions listed in the <b>Table 2</b>.</p>
                
                <p class="exp-recipe-paragraph"> Table 1 PCR reaction components and Table 2 PCR programm.</p>
                <div class="exp-recipe-image">
                    <table class="table table-responsive">
                      <thead>
                          <tr>
                              <th scope="col">Components</th>
                              <th scope="col">50 µL reaction</th>
                              <th scope="col">Final concentration</th>
                          </tr>
                      </thead>
                      <tbody>
                          <tr>
                              <td><a>Buffer 5X (HF or GC)</a></td>
                              <td>10 µL</td>
                              <td>1X</td>
                          </tr>
                          <tr>
                              <td><a>dNTPs mix (25 mM each)</a></td>
                              <td>0.4 µL</td>
                              <td>200 µM each</td>
                          </tr>
                          <tr>
                              <td><a>Forward primer (10 µM)</a></td>
                              <td>2.5 µL</td>
                              <td>0.5 µM</td>
                          </tr>
                          <tr>
                              <td><a>Reverse primer (10 µM)</a></td>
                              <td>2.5 µL</td>
                              <td>0.5 µM</td>
                          </tr>
                          <tr>
                              <td><a>DNA template: Genomic
                                DNA,Plasmid DNA</a></td>
                              <td>X µL; X µL</td>
                              <td>50 ng - 250 ng; 1 pg–10 ng</td>
                          </tr>
                          <tr>
                              <td><a>Phusion DNA Polymerase
                              </a></td>
                              <td>0.5 μL</td>
                              <td>0.02 U/µL</td>
                          </tr>
                          <tr>
                              <td><a>
                                Nuclease-free water</a></td>
                              <td>to 50 µL</td>
                              <td>-</td>
                          </tr>
                      </tbody>
                  </table>
        
                  <table class="table table-responsive table2">
                    <thead>
                        <tr>
                            <th scope="col">Step</th>
                            <th scope="col">T(°C)</th>
                            <th scope="col">Time (s)</th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td><a>Initial Denaturation</a></td>
                            <td>98</td>
                            <td>30</td>
                        </tr>
                        <tr>
                            <td rowspan="2"><a>25-35 Cycles</a></td>
                            <td>45-72</td>
                            <td>20</td>
                        </tr>
                        <tr>

                            <td>72</td>
                            <td>15-30 s per kb</td>
                        </tr>
                        <tr>
                            <td><a>Final Extension</a></td>
                            <td>72</td>
                            <td>5 min</td>
                        </tr>
                        <tr>
                            <td><a>Hold
                            </a></td>
                            <td>15</td>
                            <td>∞</td>
                        </tr>
                    </tbody>
                </table>
        
              </div>
        
                <!-- <img class="exp-recipe-ingridients-tag" src="https://2020.igem.org/wiki/images/3/36/T--Estonia_TUIT--exp_ingridients.svg">
                <ul class="exp-recipe-ingridients-list">
                    <li>DNA</li>
                    <li>T4 Ligase</li>
                    <li>T4 Buffer</li>
                    <li>H20</li>
                </ul> -->
                <!-- <h3 class="exp-recipe-header header2">Directions</h3> -->
                <!-- <p class="exp-recipe-paragraph"><b>Step 2: </b>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
                <p class="exp-recipe-paragraph"><b>Step 3: </b>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p> -->
        
            </div>
            </div>
            <div class="carousel-item">
                <div class="exp-recipe-container">
                    <h3 class="exp-recipe-header">Restriction digestion protocol</h3>
                    <p class="exp-recipe-paragraph">All restriction enzymes used in our work were FastDigest enzymes from Thermo Fisher Scientific. Restriction reactions were prepared by mixing the following reagents <b>(Table 3):</b></p>
                    
                    <div class="exp-recipe-image">
                      <table class="table table-responsive table2 table3">
                        <thead>
                            <tr>
                                <th scope="col">Components</th>
                                <th scope="col">Volumes</th>
                            </tr>
                        </thead>
                        <tbody>
                            <tr>
                                <td>Water</td>
                                <td>to 20 μL</td>
                            </tr>
                            <tr>
                                <td>10X FastDigest Buffer</td>
                                <td>2 μL</td>
                            </tr>
                            <tr>
                                <td>DNA</td>
                                <td>1 μg</td>
                            </tr>
                            <tr>
                                <td>Restriction enzymes</td>
                                <td>1 μL</td>
                            </tr>
                            <tr>
                                <td>FastAP (for vector DNA only)*</td>
                                <td>0.5 μL</td>
                            </tr>
                        </tbody>
                      </table>
                    </div>

                    <p class="exp-recipe-paragraph">*FastAP (Thermosensitive Alkaline Phosphatase) was added to the vector restriction reaction in order to prevent self-ligation of the vector without the insert.</p>
                    <p class="exp-recipe-paragraph">All components of restriction reactions were mixed in a 1.5 mL tube. After mixing, the restriction reaction was incubated at 37 °C for 30-60 min. Following the incubation, restriction enzymes were inactivated by heating the mixture for 15 min at 65-85 °C. As the next step, PCR clean-up protocol or gel electrophoresis followed by DNA purification from the gel was performed to clean the restricted DNA fragments.</p>
  
                  </div>
              </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">Agarose gel for DNA separation</h3>
                <p class="exp-recipe-paragraph">All restriction enzymes used in our work were FastDigest enzymes from Thermo Fisher Scientific. Restriction reactions were prepared by mixing the following reagents <b>(Table 3):</b></p> 
                <div class="exp-recipe-image">
                  <table class="table table-responsive table2 table3 table4">
                    <thead>
                        <tr>
                            <th scope="col">Ingredients</th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td>Agarose</td>
                        </tr>
                        <tr>
                            <td>TAE (trishydroxymethylaminomethane – ethanoic acid – ethylenediaminetetraacetic acid)
                            </td>
                        </tr>
                        <tr>
                            <td>Atlas ClearSight DNA Stain
                            </td>
                        </tr>
                    </tbody>
                  </table>
                </div>

                <ol class="exp-ol">
                  <li>In order to make 1% agarose gel, 1 g of agarose was added to 100 mL of TAE buffer solution.</li>
                  <li>The mixture was heated in a microwave until agarose was completely dissolved.</li>
                  <li>The solution was left to cool down to approximately 60 °C.
                  </li>
                  <li>
                    6 μL of Atlas ClearSight DNA Stain was added.
                  </li>
                  <li>
                    The solution was poured into a gel caster with an appropriate comb installed.
                  </li>
                  <li>
                    After polymerization, the gel was used for DNA electrophoresis.
                  </li>
                </ol>
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">Extraction of genomic DNA from yeast cells.</h3>
                <div class="exp-recipe-image">
                  <table class="table table-responsive table2 table3 table5">
                    <thead>
                        <tr>
                            <th scope="col">Ingredients</th>
                            <th scope="col">Volumes</th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td>liquid yeast culture</td>
                            <td>100-200 μL</td>
                        </tr>
                        <tr>
                            <td> lithium acetate (LiOAc), 1% SDS solution
                            </td>
                            <td>
                              100 μL of 200 mM
                            </td>
                        </tr>
                        <tr>
                            <td>96-100% ethanol
                            </td>
                            <td>
                              300 μL
                            </td>
                        </tr>
                        <tr>
                          <td>70% ethanol
                          </td>
                          <td>
                            1 mL 
                          </td>
                      </tr>
                      <tr>
                        <td>
                          H2O or TE buffer
                        </td>
                        <td>
                          100 mL
                        </td>
                      </tr>
                      <tr>
                        <td>
                          Optional: 
                        </td>
                        <td>
                          10 μL RNAse A into 50 μL of cells
                        </td>
                      </tr>
                    </tbody>
                  </table>
                </div>

                <ol class="exp-ol">
                  <li>Suspend cells in LiOAc-SDS solution
                  </li>
                  <li>Incubate 5 min 70 °C
                  </li>
                  <li>Add 96% ethanol
                  </li>
                  <li>
                    Centrifuge 15000xg 3 min, discard supernatant
                  </li>
                  <li>
                    Add 70% ethanol
                  </li>
                  <li>
                    Centrifuge 15000xg 3 min, discard supernatant
                  </li>
                  <li>
                    Dissolve the pellet in TE buffer
                  </li>
                  <li>
                    Centrifuge 15000xg 15 sec, use supernatant as template in PCR
                  </li>
                  <li>
                    Optional: Incubate with RNAse at 37 °C 1 hour
                  </li>
                </ol>
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">DNA purification from agarose gel</h3>
                <p class="exp-recipe-paragraph">All restriction enzymes used in our work were FastDigest enzymes from Thermo Fisher Scientific. Restriction reactions were prepared by mixing the following reagents <b>(Table 3):</b></p>
                
                <div class="exp-recipe-image">
                  <table class="table table-responsive table2 table3 table5">
                    <thead>
                        <tr>
                            <th scope="col">Ingredients</th>
                            <th scope="col">Volumes</th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td>liquid yeast culture</td>
                            <td>100-200 μL</td>
                        </tr>
                        <tr>
                            <td> lithium acetate (LiOAc), 1% SDS solution
                            </td>
                            <td>
                              100 μL of 200 mM
                            </td>
                        </tr>
                        <tr>
                            <td>96-100% ethanol
                            </td>
                            <td>
                              300 μL
                            </td>
                        </tr>
                        <tr>
                          <td>70% ethanol
                          </td>
                          <td>
                            1 mL 
                          </td>
                      </tr>
                      <tr>
                        <td>
                          H2O or TE buffer
                        </td>
                        <td>
                          100 mL
                        </td>
                      </tr>
                      <tr>
                        <td>
                          Optional: 
                        </td>
                        <td>
                          10 μL RNAse A into 50 μL of cells
                        </td>
                      </tr>
                    </tbody>
                  </table>
                </div>

                <ol class="exp-ol">
                  <li>In order to make 1% agarose gel, 1 g of agarose was added to 100 mL of TAE buffer solution.</li>
                  <li>The mixture was heated in a microwave until agarose was completely dissolved.</li>
                  <li>The solution was left to cool down to approximately 60 °C.
                  </li>
                  <li>
                    6 μL of Atlas ClearSight DNA Stain was added.
                  </li>
                  <li>
                    The solution was poured into a gel caster with an appropriate comb installed.
                  </li>
                  <li>
                    After polymerization, the gel was used for DNA electrophoresis.
                  </li>
                </ol>
              </div>
            </div>

<!--       KLAVS EDITS            -->
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">DNA concentration measurements</h3>
                <p class="exp-recipe-paragraph">DNA concentration measurements were carried out with NanoDrop Microvolume Spectrophotometer according to the manufacturer manuals. Software used: NanoDrop 1000 v.3.8.1.
                  Mode used: Nucleic Acid, DNA-50</p>
                  <h3 class="exp-recipe-header">Ligation protocol</h3>
                  <p class="exp-recipe-paragraph">All ligation reactions were performed using T4 DNA ligase from NEB according to the manufacturer's instructions. All the components were mixed in a 1.5 mL tube in accordance with Table 4. DNA was taken using the 1:3 ratio of vector : insert based on the size of DNA fragments.</p>
                  <p class="exp-recipe-paragraph">Table 4. Components of the ligation reaction.</p>
                  <p class="exp-recipe-paragraph">Ligation reactions were kept at 16 °C for 2-4 hours or overnight (if ligation contains blunt ends). 1-2 µL of ligation mixture was used for bacterial transformation.</p>
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">Golden Gate assembly</h3>
                <p class="exp-recipe-paragraph">Golden Gate protocol was performed using the MoClo yeast kit developed by Lee et al. (2015). The kit includes a set of characterized parts which, together with custom-made parts, allow for assembly of synthetic constructs. The Golden Gate assembly reaction was mixed together based on Table 5 and the reaction program is shown in Table 6.</p>
                <p class="exp-recipe-paragraph">Table 5. Components of Golden Gate assembly reaction</p>
                <div class="exp-recipe-image">
                  <table class="table table-responsive">
                    <thead>
                        <tr>
                            <th scope="col">Reagents</th>
                            <th scope="col">Assembly reaction</th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td><a>Plasmid or PCR product </a></td>
                            <td>0.5 μL </td>
                        </tr>
                        <tr>
                            <td><a>T4 DNA Ligase buffer (NEB) </a></td>
                            <td>1 μL</td>
                        </tr>
                        <tr>
                            <td><a>Hi-T4 DNA Ligase (NEB) </a></td>
                            <td>0.5 μL </td>
                        </tr>
                        <tr>
                            <td><a>Type IIS restriction enzyme </a></td>
                            <td>0.5 μL </td>
                        </tr>
                    </tbody>
                </table>
                </div>
                <p class="exp-recipe-paragraph">Table 6. Golden Gate reaction Thermocycler program</p>
                <div class="exp-recipe-image">
                  <table class="table table-responsive">
                    <thead>
                        <tr>
                            <th scope="col">step</th>
                            <th scope="col">Temp</th>
                            <th scope="col">Time</th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td rowspan="2"><a>25 Cycles of digestion </a></td>
                            <td>42 °C </td>
                            <td>2 min </td>
                        </tr>
                        <tr>
                            <td>16 °C </td>
                            <td>5 min</td>
                        </tr>
                        <tr>
                            <td><a>Hi-T4 DNA Ligase (NEB) </a></td>
                            <td>60 °C </td>
                            <td>10 min </td>
                        </tr>
                        <tr>
                            <td><a>Type IIS restriction enzyme </a></td>
                            <td>80 °C </td>
                            <td>10 min </td>
                        </tr>
                    </tbody>
                </table>
                </div>
              </div>
            </div>          
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">Bacterial transformation</h3>
                <p class="exp-recipe-paragraph">Necessary components:<br>
                  Escherichia coli (E. Coli) competent cells<br>
                  Lysogeny Broth (LB) media<br>
                </p>       
                <p class="exp-recipe-paragraph">DNA transformation to bacteria cells was performed as follows: </p>
                <ol class="exp-ol">
                  <li>
                  Competent Escherichia coli cells were transferred from -80 °C freezer to ice. E. coli cells were left on ice for approximately 10 min to thaw.
                  </li>
                  <li>
                  50 μL of cells was mixed with 2 μL of ligation reaction (50 pg - 1 ng for plasmid DNA).
                </li>
                  <li>
                  Cells were kept on ice for 30 min.
                </li>
                  <li>
                  Heat-shock was performed at 42 °C for 30 s.
                </li>
                  <li>
                  Cells were cooled for 1-2 min on ice.
                </li>
                  <li>
                  1 mL of LB medium was added to the cells.
                </li>
                  <li>
                  Tubes were incubated in the shaker at 37 °C and 220 rpm for 40 min.
                </li>
                  <li>
                  Meanwhile, bacterial selection plates (LB with antibiotics) were transferred from the cold room to room temperature (Plates were labeled accordingly and 10-20 glass beads were added to each of them).
                </li>
                  <li>
                  After incubation, 1.5 mL tubes with bacteria were centrifuged for 1 min at 5000 rpm.
                </li>
                  <li>
                  Excess of supernatant was discarded (approximately 250 µL was left in the tube).
                </li>
                  <li>
                  Bacterial cells were resuspended in the supernatant leftover and transferred on the selection plate.
                </li>
                  <li>
                  Bacterial cells were spread on the plate surface evenly using sterile glass beads.
                </li>
                  <li>
                  Glass beads were removed, and plates were incubated at 37 °C for 12-14 hrs (until single colonies appeared).
                  </li>
                </ol>
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">Plasmid extraction from bacterial cells</h3>
                <p class="exp-recipe-paragraph"><i>Ingredients:</i></p>
                <ul class="exp-ol">
                  <li>Minicolumns</li>
                    <li>L1 colour cell suspension solution</li>
                      <li>L2 lysis solution</li>
                        <li>GL3 neutralizing solution</li>
                          <li>W first wash solution</li>
                            <li>A1 second wash solution</li>
                              <li>TE buffer</li>
                  </ul>

                  <p class="exp-recipe-paragraph"> Plasmids were extracted from bacterial cells using the “Plasmid Mini” kit by A&A Biotechnology according to manual instructions. A brief description of the procedure is written below.  </p>
                  <ol class="exp-ol">
                    <li>
                    Up to 3 mL (1.5-3 mL) of overnight bacterial culture was centrifuged. After that supernatant was removed and the bacterial pellets were suspended in 200 µL of L1 cell suspension solution.
                    </li>
                    <li>
                    200 µL of L2 lysis solution was added and the solution was carefully mixed by turning the 1.5 mL tube up-side-down. The mix was incubated for 3 min at room temperature.
                    </li>
                    <li>
                    400 µL of GL3 neutralizing solution was added and carefully mixed until the disappearance of the raspberry color of the lysates.
                    </li>
                    <li>
                    The lysates were centrifuged for 10 min at 10 000-15 000 RPM.
                    </li>
                    <li>
                    The supernatants were transferred onto the minicolumns, which were centrifuged for 1 min at 10 000-15 000 RPM.
                    </li>
                    <li>
                    The minicolumns were removed from the collection tubes and the filtrates were discarded then the minicolumns were re-assembled with the same collection tubes.
                    </li>
                  </ol>
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
            <ol class="exp-ol" start="7">
            <li>
              500 µL of W first wash solution were added and the minicolumns were centrifuged for 1 min at 10 000-15 000 RPM.
              </li>
              <li>
              The minicolumns were removed from the collection tubes, the filtrates were discarded and the minicolumns re-assembled with the same collection tubes.
              </li>
              <li>
              600 µL of A1 second wash solution were added and centrifugation for 2 min at 10 000-15 000 RPM was performed.
              </li>
              <li>
              The dry minicolumns were carefully transferred into new 1.5 mL tubes. 60 µL of TE buffer was added directly onto the minicolumns resin to elute DNA.
              </li>
              <li>
              Minicolumns were incubated for 3 min at room temperature followed by 1 min centrifugation at 12 000 RPM.
              </li>
              <li>
              Eluted plasmid DNA was used straight away or stored at -20°C until needed.
              </li>
            </ol>
            <p class="exp-recipe-paragraph"> <b>Additional protocols used:</b> <br>
              <a color= "black" href="http://www.favorgen.com/favorgen/serv_1/mem_t1/h_1/pdf/plasmid/FAPDE%20004%20100%20300.pdf">FavorPrep<sup>TM</sup> Plasmid Extraction Mini Kit </a><br>
              <a color= "black" href="https://international.neb.com/-/media/nebus/files/manuals/manualt1010.pdf?rev=5bc17659c6df4f9cbb585e6d6973180d&hash=926E0752DD30125061D3419C486176A6">Monarch<sup>TM</sup> Plasmid Miniprep Kit </a><br>
            </p>
            </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">Plasmid rescue from yeast</h3>
                <p class="exp-recipe-paragraph"><i>Ingredients:</i></p>
                <ul class="exp-ol">
                  <li>STET buffer</li>
                  <li>Ammonium acetate</li>
                  <li>96% ethanol</li>
                  <li>70% ethanol</li>
                  <li>MQ water</li>
                </ul>
                <p class="exp-recipe-paragraph"><i>Plasmid extraction from yeast was performed following protocol written bellow:</i></p>
                <ol class="exp-ol">
                <li>
                Grow a 1.5 ml culture overnight under selective conditions.
                </li>
                <li>
                Harvest cells by centrifugation (6000 rpm, 1 minute).
                </li>
                <li>
                Resuspend the pellet in 100 µl  of STET buffer and add 0.2 g of 0.45 mm glass beads.
                </li>
                <li>
                Vortex the tubes vigorously for 5 minutes or use bead-beater (4.0 m/s, 40 seconds).
                </li>
                <li>
                Add another 10 µl of STET and vortex the tubes briefly.
                </li>
                <li>
                Incubate tube at 100 °C for 3 minutes.
                </li>
                <li>
                Cool briefly on ice and spin in a centrifuge for 10 minutes at 4 °C at max RPM.
                </li>
                <li>
                Transfer 100 µl of the supernatant to a new tube containing 50 µl of 7.5 M ammonium acetate.
                </li>
                <li>
                Incubate at -20 °C for 1 hour.
                </li>
                <li>
                Centrifuge for 10 minutes at 4 °C max RPM.
                </li>
                <li>
                Add 100 µl of supernatant to 200 µl of ice-cold 96% ethanol and recover the DNA by centrifugation at max RPM.
                </li>
                <li>
                Wash with 1000 µL of 70% ethanol and resuspend pellet in 20 µl of MQ water.
                </li>
                </ol>
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
            <ol class="exp-ol" start="13">
            <li>
              Use 10 µl of this solution to transform competent bacteria.
            </li>
            </ol>

              <p class="exp-recipe-paragraph"><i>STET buffer mentioned in this protocol was prepared according to such proportions:</i></p>
              <ul class="exp-ol">
                <li>8% sucrose;</li>
                <li>50 mM Tris-HCl, pH 8.0;</li>
                <li>50 mM EDTA;</li>
                <li>70% ethanol</li>
                <li>5% Triton X-100.</li>
              </ul>
              </div>
              </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">Yeast transformation</h3>
                <p class="exp-recipe-paragraph"><i>Ingredients:</i></p>
                <ul class="exp-ol">
                  <li>YPD (yeast extract peptone dextrose) medium</li>
                  <li>100 mM LiAc (Lithium Acetate)</li>
                  <li>TE buffer (tris(hydroxymethyl)aminomethane - Ethylenediaminetetraacetic acid buffer)</li>
                  <li>Salmon Sperm DNA (SS-DNA)</li>
                  <li>PEG (Poly(ethylene glycol))/LiAc (40% PEG 3350 + 100mM LiAc in 1 x TE)</li>
                  <li>DMSO</li>
                </ul>
                <p class="exp-recipe-paragraph">In order to genetically transform yeast cells, the desired yeast strain was pre-grown in 50 mL of liquid YPD until OD600 0.6-0.8. The transformation was performed based on the following protocol:</p>
                <ol class="exp-ol">
                  <li>
                  Cell culture was centrifuged in a sterile 50 mL falcon tube. The supernatant was discarded.
                  </li>
                  <li>
                  The cell pellet was resuspended in 1 mL of sterile 100 mM LiAc in TE and transferred into 1.5 mL tube.
                  </li>
                  <li>
                  Cells were centrifuged at 3600 rpm for 60 s. The supernatant was removed and cells were resuspended in two times of cell volume of 100 mM LiAc in TE.
                  </li> 
                  <li>
                  Cells were incubated for 10 min at room temperature.
                  </li>  
                  <li>
                  Single-stranded carrier DNA was prepared: Salmon Sperm DNA(SS-DNA) was boiled for 10 min at 100 °C and immediately chilled on ice before use. This ensured that the DNA was single-stranded. SS-DNA was kept on ice for the rest of the time before use.
                  </li>
                  <li>
                  20 μL of plasmid DNA (yeast integration vector restricted with proper restriction enzyme) and 10 μL of SS-DNA was mixed in a separate 1.5 mL tube.
                  </li>
                </ol>
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
            <ol class="exp-ol" start="7">
            <li>
              100 μL of yeast competent cells were added to the DNA and mixed.
              </li>
              <li>
              At the next step, 700 ul of sterile PEG/LiAc (40% PEG 3350 + 100mM LiAc in 1 x TE) were added.
              </li>
              <li>
              Finally, 48 μL of DMSO ((CH3)2SO) was added and everything was gently mixed by pipetting.
              </li>
              <li>
              The mixture was incubated at 42 °C for 40 min (heat shock).
              </li>
              <li>
              After the heat shock, the cells were chilled on ice for 2 min.
              </li>
              <li>
              At the next step, the cells were centrifuged for 30 s at 6000 rpm at room temperature. The supernatant was removed and the cells were resuspended in 1 mL of sterile 1x TE buffer.
              </li>
              <li>
              After that, the cells were centrifuged for 60 s at 3600 rpm at room temperature. The supernatant was discarded and the cells were resuspended in 200 μL of sterile 1x TE buffer.
              </li>
              <li>
              Finally, 200 μL of suspension were transferred on the yeast selection plates and cells were evenly spread all over the plate surface using sterile glass beads.
              </li>
              <li>
              The plates were incubated upside down at 30 °C until colonies appeared (usually, for 2-3 days).
              </li>
            </ol>
            </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">Yeast colony PCR</h3>
                <p class="exp-recipe-paragraph">Preparation of yeast cell lysates:</p>
                <ol class="exp-ol">
                  <li>Cells of a single yeast colony (taken with a pipette tip) were transferred into 30 µL of 20 mM NaOH in a 1.5 mL tube.</li>
                  <li>The mixture was boiled at 100 °C for 10 min.</li>
                  <li>Lysates were placed on ice for cooling followed by 1 min centrifugation at 12000 rpm.</li>
                  <li>Yeast cell lysates were placed on ice.</li>
                  <li>1 µL of supernatant was used as a template for PCR reaction.</li>
                </ol>
                <p class="exp-recipe-paragraph">Preparation of PCR mixture</p>
                <p class="exp-recipe-paragraph">Components of the PCR reaction were mixed according to <b>Table 7.</b></p>
                
                <div id="dualtable" class="exp-recipe-image">
                  <div id="table1">
                    <table class="table table-responsive">
                      <thead>
                          <tr>
                              <th scope="col">Step</th>
                              <th scope="col">Temp</th>
                              <th scope="col">Time</th>
                          </tr>
                      </thead>
                      <tbody>
                          <tr>
                              <td rowspan="2">25 cycles of digestion</td>
                              <td>42 °C</td>
                              <td>2 min</td>
                          </tr>
                          <tr>
                              <td>16 °C</td>
                              <td>5 min</td>
                          </tr>
                          <tr>
                              <td>Final digestion</td>
                              <td>60 °C</td>
                              <td>10 min</td>
                          </tr>
                          <tr>
                              <td>Heat inactivation</td>
                              <td>80 °C</td>
                              <td>10 min</td>
                          </tr>
                      </tbody>
                    </table>
                    <p class="exp-recipe-paragraph"><b>Table 7. Components for PCR with “PCR Mix Plus Green” (Provided by A&A Biotechnology).</b></p>
                  </div>
                  <div>
                    <table class="table table-responsive">
                      <thead>
                          <tr>
                              <th scope="col">Components</th>
                              <th scope="col">Volumes (°C)</th>
                          </tr>
                      </thead>
                      <tbody>
                          <tr>
                              <td>Water</td>
                              <td>5 µL</td>
                          </tr>
                          <tr>
                              <td>2X "PCR Mix Plus Green"</td>
                              <td>12.5 µL</td>
                          </tr>
                          <tr>
                              <td>DNA template</td>
                              <td>1 µL</td>
                          </tr>
                          <tr>
                              <td>Forward & reverse primers</td>
                              <td>0.75 + 0.75 μL</td>
                          </tr>
                      </tbody>
                    </table>
                    <p class="exp-recipe-paragraph"><b>Table 8. PCR program.</b></p>
                  </div>

                </div>              
                <p class="exp-recipe-paragraph">Conditions used for yeast colony PCR are described in <b>Table 8.</b></p>
                <div class="exp-recipe-image">

                </div> 
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">LB and YPD media</h3>
                <p class="exp-recipe-paragraph">Tables 9 and 10, respectively, provide list of components for preparation of LB and YPD media.</p>
                <p class="exp-recipe-paragraph"><b>Table 9. LB medium components.</b></p>
                <div id="dualtable" class="exp-recipe-image">
                  <div id="table1">
                <div class="exp-recipe-image">
                  <table class="table table-responsive">
                    <thead>
                        <tr>
                            <th scope="col">Components</th>
                            <th scope="col">Amounts</th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td>Tryptone</td>
                            <td>10 g</td>
                        </tr>
                        <tr>
                            <td>Yeast extract</td>
                            <td>5 g</td>
                        </tr>
                        <tr>
                            <td>NaCl</td>
                            <td>10 g</td>
                        </tr>
                        <tr>
                            <td>H<sub>2</sub>O</td>
                            <td>up to 1L</td>
                        </tr>
                    </tbody>
                </table>
                </div>
            </div>
            <div table="table2">
                <p class="exp-recipe-paragraph"><b>Table 10. YPD components.</b></p>
                <div class="exp-recipe-image">
                  <table class="table table-responsive">
                    <thead>
                        <tr>
                            <th scope="col">Components</th>
                            <th scope="col">Amounts</th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td>Peptone</td>
                            <td>20 g/L</td>
                        </tr>
                        <tr>
                            <td>D-Glucose anhydrous</td>
                            <td>20 g/L</td>
                        </tr>
                         <tr>
                            <td>Micro granulated yeast extract</td>
                            <td>10 g/L</td>
                        </tr>
                        <tr>
                            <td>Bacto agar (for plates)</td>
                            <td>15 g/L</td>
                        </tr>
                    </tbody>
                </table>
                </div>
                  </div>
                </div>
                <ol class="exp-ol">
                  <li>All components were mixed in 2 L Erlenmeyer flask with a magnetic stirrer in it.</li>
                  <li>15 g of Bacto Agar (for plates only, Bacto Agar was omitted for liquid medium) was added.</li>
                  <li>The medium was autoclaved for 15 min at 121 °C.</li>
                  <li>The medium was left to cool down to approximately 55 °C.</li>
                  <li>If necessary, 1 mL of sterile 100 mg/mL antibiotic solution was added to the medium.</li>
                  <li>Approx. 30 mL of medium was poured into the sterile Petri dishes and was left for cooling down and polymerization.</li>
                  <li>Plates were kept at +4°C for up to two weeks.</li>
                </ol>
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">Nile red staining</h3>
                <p class="exp-recipe-paragraph">Nile red is a lipophilic stain which can be used as a lipid droplet dye and can be visualized under a fluorescence microscope. Nile red staining was performed as described by Greenspan et al. (1985). </p>
                <p class="exp-recipe-paragraph"><i>Ingredients:</i><br>
                  1 mg/mL Nile red in DMSO<br>
                  Phosphate-buffered saline (PBS)<br>
                </p>

                <div class="exp-recipe-image">
                <ol class="exp-ol">
                  <li>Centrifuge 100 µL of cell culture at 2000 x g for 1 minute</li>
                  <li>Remove the supernatant and resuspend the pellet in 500 µL PBS. </li>
                  <li>Add 5 µL 1 mg/mL Nile red solution in DMSO to the cell suspension.</li>
                  <li>Centrifuge the cells at 2000 x g and wash twice with 1 mL PBS. </li>
                  <li>Resuspend in 100 µL PBS. </li>
                </ol>
                </div>
                <h3 class="exp-recipe-header">Microscopy</h3>
                <p class="exp-recipe-paragraph">For the promoter characterization part, yeast colonies were checked after the transformation for the presence of GFP fluorescence. Prior to the experiment, yeast cultures were grown in synthetic complete uracil dropout medium with 2% glucose as a carbon source. The optical density (OD600) was in the range 0.2-0.6. Cultures were imaged under a Zeiss Axio Observer Z1 microscope. The following settings were used for the imaging: 25 ms exposure time for brightfield or phase-contrast, 100 ms exposure time for EGFP channel. Colibri LED modules were used as a source of 470 nm excitation at 25% intensity. Nile red was excited with the 470 nm LED for 5 ms. The fluorescence intensities were quantified using ImageJ.</p>
             </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">Time-lapse microscopy</h3>
                <p class="exp-recipe-paragraph">Prior to time-lapse microscopy, the yeast cells were grown in synthetic complete (SC) media lacking histidine or SC lacking histidine and tryptophan depending on the integrative plasmids in the strain. The cultures were grown to OD600 0.2-0.8, the cells were pipetted onto a 0.08 mm cover glass slip and were covered with 1.5% agar-SC. Zeiss Observer Z1 microscope with 63C/1.4NA oil immersion objective and Axiocam 506 mono camera was used for imaging. The focus was kept using Definite Focus and the sample was kept at 30 °C using PeCon TempControl 37-2 digital. The cells were imaged every 5 minutes using brightfield illumination and the experiments were 13-18 hours long.</p>
                <h3 class="exp-recipe-header">Plate reader</h3>
                <p class="exp-recipe-paragraph">Plate readers are instruments that are used to detect samples in microtiter plates. We used BioTek Synergy MX microplate reader to measure the cells’ optical density and intensity of EGFP fluorescence. All strains used in the experiments were pre-grown overnight at 30 °C in liquid CSM/2%Glc media and diluted to OD600 in a range of 0.9 to 1.1. Then diluted cell cultures were distributed to 96 well plates (clear flat bottom). For replicates of two colonies from each strain were analyzed. 200 µL of cells were added to each well. CSM/2%Glc media was used as a reference to OD600. The measurements were done with the following parameters. Absorbance was measured at 600 nm and fluorescence was measured at the following conditions: excitation 485 nm, emission 528 nm, bandwidth 20, gain 80.</p>
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">InP Nanoparticles activation</h3>
                <p class="exp-recipe-paragraph"><i>Ingredients:</i><br>
                  MQ water<br>
                  InP piece<br>
                </p>

                <div class="exp-recipe-image">
                <ol class="exp-ol">
                  <li>
                  Hammer the InP into small pieces.
                  </li>
                  <li>
                  Grind with mortar and pestle in water solution until you do not feel any particles with the mortar (ca. 2h for around 2.5 g piece).
                  </li>
                  <li>
                  Transfer this nanoparticle water suspension into two 5 ml tubes.
                  </li>
                  <li>
                  Ultrasonicate to separate particles that have aggregated. 
                  </li>
                  <li>
                  Dilute in 50 ml of distilled water.
                  </li>
                  <li>
                  Centrifuge at 8000x g for 5 min.
                  </li>
                  <li>
                  Big particles remain on the walls of the flask, the suspension is transferred into a new flask.
                  </li>
                  <li>
                  Centrifuge at 8000x g for 5 min.
                  </li>
                  <li>
                  Resuspend the remaining particles from step 7 in 50 ml distilled water. Sonicate and centrifuge at 8000x g for 5 min. Collect both supernatants from steps 8 and 9 into a new tube.
                  </li>
                  <li>
                  Centrifuge at 10000x g for 15 min.
                  </li>
                  <li>
                  Transfer suspension into a new bigger tube and add additional 50 ml of water.  
                  </li>
                  <li>
                  Centrifuge at 8000x g for 5 min.
                  </li>
                  <li>
                  Transfer suspension into a new tube.
                  </li>
                  <li>
                  Centrifuge at 10000x g for 15 min.
                  </li>
                  <li>
                  Transfer suspension into a new tube.
                  </li>
                  <li>
                  Centrifuge at 8000x g for 5 min.
                  </li>
                  <li>
                  Transfer suspension into a new tube.
                  </li>
                  <li>
                  Centrifuge at 10000x g for 15 min
                  </li>
                  <li>
                  The nanoparticles are now on the tube wall, and are collected by wall scraping, and further transferred into 5 ml tube and resuspended in distilled water.
                  </li>
                </ol>
                </div>
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">Activation of nanoparticles</h3>
                <p class="exp-recipe-paragraph"><i>Ingredients:</i><br>
                  MQ water<br>
                  FeCl3∙6H2O (5 mg mL-1)<br>
                  Tannic acid (40 mg mL-1)<br>
                  Tris-HCl buffer (pH 8.0, 1M)<br>
                  70% ethanol<br>                
                </p>
                <div class="exp-recipe-image">
                <ol class="exp-ol">
                  <li>
                  Suspend nanoparticles in MQ water to 0.5 – 5% w/v.
                  </li>
                  <li>
                  Transfer 500 µL of suspension into a new 2 ml tube.
                  </li>
                  <li>
                  Add 500 µL of FeCl3∙6H2O (5 mg mL-1) and 500 µL of tannic acid (40 mg mL-1). Mix by pipetting up and down.
                  </li>
                  <li>
                  Add 60 µL of Tris buffer (pH 8.0, 1M). Slowly add it till pH of the solution becomes 8!
                  </li>
                  <li>
                  Spin down nanoparticles at 8,000 g, 5 min.
                  </li>
                  <li>
                  Remove supernatant. Wash the pellet 3-4 times with MQ water at 8,000 g, 5 min.
                  </li>
                  <li>
                  After the wash, add 1 mL of 70% ethanol and incubate at room temperature for 10 min.
                  </li>
                  <li>
                  Centrifuge at 8,000 g, 5 min. Repeat step 6 in full.
                  </li>
                  <li>
                  After final wash, resuspend nanoparticles in 1 mL MQ water.
                  </li>
                  <li>
                  Sonicate the suspension before use.
                  </li>
                </ol>
                </div>
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">Cell surface modification</h3>
                <p class="exp-recipe-paragraph"><i>Ingredients:</i>
                <br>
                MQ water<br>
                Polyallylamine hydrochloride (PAH) (5 mg mL-1)<br>
                </p>

                <ol class="exp-ol">
                  <li>
                  Wash the grown cell culture in MQ water 3-4 times. Centrifugation is performed at 3,000g, 2 min.
                  </li>
                  <li>
                  Resuspend cells in 500 µL of MQ water to achieve OD600 1.6 ~ 3.2.
                  </li>
                  <li>
                  Dilute and transfer 450 µL of cell suspension. Recommended OD600 1.6 ~ 3.2.
                  </li>
                  <li>
                  Add 50 µL of PAH (polyallylamine hydrochloride) and gently vortex for 20 ~ 40 seconds before spinning down and removing the supernatant.
                  </li>
                  <li>
                  Wash the pellet in MQ water 3-4 times. Centrifugation is performed at 3,000g, 2 min.
                  </li>
                  <li>
                  Resuspend yeast in MQ water to achieve desirable OD600.
                  </li>
                </ol>
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">Cell surface modification</h3>
                <p class="exp-recipe-paragraph"><i>Ingredients:</i>
                <br>
                MQ water<br>
                Polyallylamine hydrochloride (PAH) (5 mg mL-1)<br>
                </p>

                <ol class="exp-ol">
                  <li>
                  Wash the grown cell culture in MQ water 3-4 times. Centrifugation is performed at 3,000g, 2 min.
                  </li>
                  <li>
                  Resuspend cells in 500 µL of MQ water to achieve OD600 1.6 ~ 3.2.
                  </li>
                  <li>
                  Dilute and transfer 450 µL of cell suspension. Recommended OD600 1.6 ~ 3.2.
                  </li>
                  <li>
                  Add 50 µL of PAH (polyallylamine hydrochloride) and gently vortex for 20 ~ 40 seconds before spinning down and removing the supernatant.
                  </li>
                  <li>
                  Wash the pellet in MQ water 3-4 times. Centrifugation is performed at 3,000g, 2 min.
                  </li>
                  <li>
                  Resuspend yeast in MQ water to achieve desirable OD600.
                  </li>
                </ol>
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">Microbial biohybrid assembly</h3>
                <p class="exp-recipe-paragraph"><i>Ingredients:</i>
                <br>
                FeCl3∙6H2O (5 mg mL-1)<br>
                PBS (pH 7.4)<br>
                MQ water<br>
                </p>

                <ol class="exp-ol">
                  <li>
                  Use OD600 cell to nanoparticles ratio 1.6 ~ 2.0 for the assembly.
                  </li>
                  <li>
                  Mix cells with nanoparticles in a new 1.5 mL tube. Vortex mix for 10-60 s.
                  </li>
                  <li>
                  Add FeCl3∙6H2O (5 mg mL-1) in a relation of 0.01 : 1 to full volume of mix in step 2.
                  </li>
                  <li>
                  Add an equal amount of PBS (pH 7.4) as volume of mix from step 2.
                  </li>
                  <li>
                  Wash final mix with MQ water 3 times at 2,000 g, 2 min. 
                  </li>
                  <li>
                  Transfer mix into selected growth medium.
                  </li>
                </ol>
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <h3 class="exp-recipe-header">CRISPR/Cas9 protocol </h3>
                <p class="exp-recipe-paragraph">We used the Ellis Lab CRISPR/Cas9 toolkit as described by William Shaw. This method allows for markerless integration of multiple genetic cassettes into selected genomic loci. The toolkit consists of: 
                <br>
                An sgRNA entry vector<br> 
                A range of Cas9-sgRNA expression vectors<br> 
                Markerless integration cassettes <br>
                </p>
                <p class="exp-recipe-paragraph">gRNAs targeting ACC1 region were designed using the Benchling CRISPR Guide RNA design tool. Four gRNAs with the highest on- and lowest off-target scores were chosen, and 5’ overhangs were added for cloning into a BsmBI-linearized plasmid. The following step describes the process of gRNA assembly and cloning. 
                </p>
                <p class="exp-recipe-paragraph">Phosphorylation of oligos (inserts need to have 5’ phosphates for ligation, but synthesized primers do not have this).  </p>
                  <div class="exp-recipe-image">
                    <table class="table table-responsive">
                      <thead>
                          <tr>
                              <th scope="col">Components</th>
                              <th scope="col">Volumes</th>
                          </tr>
                      </thead>
                      <tbody>
                          <tr>
                              <td>10x T4 DNA ligase buffer 
                  (Thermo Scientific)</td>
                              <td>1 µL</td>
                          </tr>
                          <tr>
                              <td>sgRNA oligo 100 µM</td>
                              <td>1 µL</td>
                          </tr>
                           <tr>
                              <td>T4 PNK</td>
                              <td>1 µL</td>
                          </tr>
                          <tr>
                              <td>H<sub>2</sub>O</td>
                              <td>7 µL</td>
                          </tr>
                      </tbody>
                  </table>
                  </div>
                <p class="exp-recipe-paragraph">
                  The mixture was incubated 1h at 37 °C for phosphorylation reaction. Then PNK was inactivated at 75 °C for 10 min. 
                </p>
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                <ol class="exp-ol">
                  <li>
                    For annealing of oligos, 10 µL of each of the two of phosphorylated oligos were mixed and 180 µL of MQ H2O was added. 50 µL of this mix was run in the thermocycler with the following program: 96 °C for 6 min, followed by a decrease in increments of 0.1 °C per second, down to 23 °C. 
                  </li>
                  <li>
                    For ligation of the annealed oligos to sgRNA entry vector, the following mix was prepared and kept at room temperature for 15 min. The resulting mix was  
                  </li>  
                  </ol>
                  <div id="dualtable" class="exp-recipe-image">
                    <div id="table1">
                  <div class="exp-recipe-image">
                    <table class="table table-responsive">
                      <thead>
                          <tr>
                              <th scope="col">Components</th>
                              <th scope="col">Volumes</th>
                          </tr>
                      </thead>
                      <tbody>
                          <tr>
                              <td>Annealed oligos</td>
                              <td>0.5 µL</td>
                          </tr>
                          <tr>
                              <td>BsmBI-linearized pWS082</td>
                              <td>0.5 µL</td>
                          </tr>
                          <tr>
                              <td>10x T4 ligase buffer (Thermo Scientific)</td>
                              <td>1 µL</td>
                          </tr>
                          <tr>
                              <td>T4 DNA Ligase</td>
                              <td>0.5 µL</td>
                          </tr>
                          <tr>
                              <td>H<sub>2</sub>O</td>
                              <td>7.5 µL</td>
                          </tr>
                      </tbody>
                  </table>
                  </div>
                  </div>
                  <div class="table2">
                  <div class="exp-recipe-image">
                    <table class="table table-responsive">
                      <thead>
                          <tr>
                              <th scope="col">Components</th>
                              <th scope="col">Volumes</th>
                          </tr>
                      </thead>
                      <tbody>
                          <tr>
                              <td>sgRNA vector</td>
                              <td>1 µg</td>
                          </tr>
                          <tr>
                              <td>10x FastDigest buffer</td>
                              <td>1 µL</td>
                          </tr>
                          <tr>
                              <td>Eco32I</td>
                              <td>0.5 µL</td>
                          </tr>
                          <tr>
                              <td>H<sub>2</sub>O</td>
                              <td>Up to 10 µL</td>
                          </tr>
                      </tbody>
                  </table>
                  </div>
                  </div>
                  </div>
                  <p class="exp-recipe-paragraph">Once assembled and sequence verified, the sgRNA expression cassette was linearized with FastDigest Eco32I. </p>
                  <p class="exp-recipe-paragraph"> The mix was incubated at 37 °C for 30 min. </p>
                  
              </div>
            </div>
            <div class="carousel-item">
              <div class="exp-recipe-container">
                
                <p class="exp-recipe-paragraph">
                  For genome editing, the following mix was prepared and used in the Super-High Efficiency yeast transformation protocol provided by Ellis Lab. 
                </p>
                <div class="exp-recipe-image">
                    <table class="table table-responsive">
                      <thead>
                          <tr>
                              <th scope="col">Components</th>
                              <th scope="col">Transformation reaction</th>
                          </tr>
                      </thead>
                      <tbody>
                          <tr>
                              <td>Linearized Cas9-gRNA gaprepair vector</td>
                              <td>100 ng</td>
                          </tr>
                          <tr>
                              <td>Each digested sgRNA vector</td>
                              <td>200 ng</td>
                          </tr>
                          <tr>
                              <td>PCR amplified donor DNA</td>
                              <td>> 2 µg </td>
                          </tr>
                      </tbody>
                  </table>
                </div>
                <p class="exp-recipe-paragraph">
                  For transformation, we used 2 µL of the linearized Cas9-sgRNA repair vector (pWS171), 2 µL of linearized sgRNA cassette, and 36 µL of ACC1S686A S659A S1157A purified PCR product
                </p>

                <h3 class="exp-recipe-header">Yeast viability assay</h3>
                <p class="exp-recipe-paragraph">
                  The cultures were diluted to 106 cells/mL. Then, this culture was diluted 2, 10, 100 and 500 times. 5 µL of the dilution series was plated on YPD plates. The plated were incubated overnight at 30 °C.
                </p>
              </div>
            </div>
          </div>
          <a class="carousel-control-prev" href="#experimentCarousel" role="button" data-slide="prev">
              <span class="carousel-control-prev-icon arrow-static" aria-hidden="true"></span>
              <span class="sr-only">Previous</span>
            </a>
            <a class="carousel-control-next" href="#experimentCarousel" role="button" data-slide="next">
              <span class="carousel-control-next-icon arrow-static" aria-hidden="true"></span>
              <span class="sr-only">Next</span>
            </a>
        </div>
    </div>
  </section>

  <section class="experiments-section">
    <img class="exp-bg" src="https://2020.igem.org/wiki/images/6/65/T--Estonia_TUIT--bg_exp.svg">
    <img class="exp-molecule1" src="https://2020.igem.org/wiki/images/c/cb/T--Estonia_TUIT--exp_molecule1.svg">
    <img class="exp-penpaper" src="https://2020.igem.org/wiki/images/3/3c/T--Estonia_TUIT--exp_paper_pen.svg">
    <img class="exp-flasks" src="https://2020.igem.org/wiki/images/6/66/T--Estonia_TUIT--exp_flasks.svg">
    <img class="exp-molecule2" src="https://2020.igem.org/wiki/images/c/cb/T--Estonia_TUIT--exp_molecule1.svg">
    <img class="exp-molecule3" src="https://2020.igem.org/wiki/images/7/71/T--Estonia_TUIT--exp_molecule2.svg">
  </section>

  
  <div class="footer">
    <div class="contactblock">
      <p>Contact Us: tartutuit@gmail.com</p>
      <div class="social">
        <p>Facebook:</p>
        <svg
          id="facebook"
          enable-background="new 0 0 24 24"
          viewBox="0 0 24 24"
          width="70"
        >
        <path
          fill="#FFF"
          d="m15.997 3.985h2.191v-3.816c-.378-.052-1.678-.169-3.192-.169-3.159 0-5.323 1.987-5.323 5.639v3.361h-3.486v4.266h3.486v10.734h4.274v-10.733h3.345l.531-4.266h-3.877v-2.939c.001-1.233.333-2.077 2.051-2.077z"
        />
        </svg>
        <p>Instagram:</p>
        <svg class="instagram" viewBox="0 0 511 511.9" width="60">
          <path
            fill="#FFF"
            d="m510.949219 150.5c-1.199219-27.199219-5.597657-45.898438-11.898438-62.101562-6.5-17.199219-16.5-32.597657-29.601562-45.398438-12.800781-13-28.300781-23.101562-45.300781-29.5-16.296876-6.300781-34.898438-10.699219-62.097657-11.898438-27.402343-1.300781-36.101562-1.601562-105.601562-1.601562s-78.199219.300781-105.5 1.5c-27.199219 1.199219-45.898438 5.601562-62.097657 11.898438-17.203124 6.5-32.601562 16.5-45.402343 29.601562-13 12.800781-23.097657 28.300781-29.5 45.300781-6.300781 16.300781-10.699219 34.898438-11.898438 62.097657-1.300781 27.402343-1.601562 36.101562-1.601562 105.601562s.300781 78.199219 1.5 105.5c1.199219 27.199219 5.601562 45.898438 11.902343 62.101562 6.5 17.199219 16.597657 32.597657 29.597657 45.398438 12.800781 13 28.300781 23.101562 45.300781 29.5 16.300781 6.300781 34.898438 10.699219 62.101562 11.898438 27.296876 1.203124 36 1.5 105.5 1.5s78.199219-.296876 105.5-1.5c27.199219-1.199219 45.898438-5.597657 62.097657-11.898438 34.402343-13.300781 61.601562-40.5 74.902343-74.898438 6.296876-16.300781 10.699219-34.902343 11.898438-62.101562 1.199219-27.300781 1.5-36 1.5-105.5s-.101562-78.199219-1.300781-105.5zm-46.097657 209c-1.101562 25-5.300781 38.5-8.800781 47.5-8.601562 22.300781-26.300781 40-48.601562 48.601562-9 3.5-22.597657 7.699219-47.5 8.796876-27 1.203124-35.097657 1.5-103.398438 1.5s-76.5-.296876-103.402343-1.5c-25-1.097657-38.5-5.296876-47.5-8.796876-11.097657-4.101562-21.199219-10.601562-29.398438-19.101562-8.5-8.300781-15-18.300781-19.101562-29.398438-3.5-9-7.699219-22.601562-8.796876-47.5-1.203124-27-1.5-35.101562-1.5-103.402343s.296876-76.5 1.5-103.398438c1.097657-25 5.296876-38.5 8.796876-47.5 4.101562-11.101562 10.601562-21.199219 19.203124-29.402343 8.296876-8.5 18.296876-15 29.398438-19.097657 9-3.5 22.601562-7.699219 47.5-8.800781 27-1.199219 35.101562-1.5 103.398438-1.5 68.402343 0 76.5.300781 103.402343 1.5 25 1.101562 38.5 5.300781 47.5 8.800781 11.097657 4.097657 21.199219 10.597657 29.398438 19.097657 8.5 8.300781 15 18.300781 19.101562 29.402343 3.5 9 7.699219 22.597657 8.800781 47.5 1.199219 27 1.5 35.097657 1.5 103.398438s-.300781 76.300781-1.5 103.300781zm0 0"
          />
          <path
            fill="#FFF"
            d="m256.449219 124.5c-72.597657 0-131.5 58.898438-131.5 131.5s58.902343 131.5 131.5 131.5c72.601562 0 131.5-58.898438 131.5-131.5s-58.898438-131.5-131.5-131.5zm0 216.800781c-47.097657 0-85.300781-38.199219-85.300781-85.300781s38.203124-85.300781 85.300781-85.300781c47.101562 0 85.300781 38.199219 85.300781 85.300781s-38.199219 85.300781-85.300781 85.300781zm0 0"
          />
          <path
            fill="#FFF"
            d="m423.851562 119.300781c0 16.953125-13.746093 30.699219-30.703124 30.699219-16.953126 0-30.699219-13.746094-30.699219-30.699219 0-16.957031 13.746093-30.699219 30.699219-30.699219 16.957031 0 30.703124 13.742188 30.703124 30.699219zm0 0"
          />
        </svg>
      </div>
    </div>
    <div class="sponsorblock">
      <img src="https://2020.igem.org/wiki/images/8/87/T--Estonia_TUIT--BioLabsW.png" >
      <img src="https://2020.igem.org/wiki/images/3/39/T--Estonia_TUIT--ecsbW.png" >
      <img src="https://2020.igem.org/wiki/images/b/b0/T--Estonia_TUIT--idtW.png" >
      <img src="https://2020.igem.org/wiki/images/2/29/T--ESTONIA_TUIT--MathWorks.png" >
      <img src="https://2020.igem.org/wiki/images/b/ba/T--Estonia_TUIT--tuitW.png" >
    </div>
  </div>

  <script src="node_modules/jquery/dist/jquery.slim.min.js"></script>
  <script type="module" src="assets/js/bootstrap.min.js"></script>
</body>

</html>